src/getHumanAge.test.js

```diff
@@ -7,4 +7,32 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('when input 0 values, must return 0', () => {
```

Comment:
```suggestion
  test('should return [0, 0] when cat and dogs years are equal to 0, ', () => {
```

smth like that, because now it seems like the function returns a number, but it's not true